bin/postgres-ha/bootstrap/pre-bootstrap.sh

```diff
@@ -322,6 +293,17 @@ build_bootstrap_config_file() {
         /opt/cpm/bin/yq m -i -x "${bootstrap_file}" "/opt/cpm/conf/postgres-ha-initdb.yaml"
     fi
 
+    if [[ -n "${PGHA_WALDIR}" ]]
+    then
+        echo_info "Applying custom WAL dir to postgres-ha configuration"
+        if printf '10\n'${PGVERSION} | sort -VC
+        then
+            /opt/cpm/bin/yq w -i "${bootstrap_file}" 'bootstrap.initdb[+].waldir' "${PGHA_WALDIR}"
+        else
+            /opt/cpm/bin/yq w -i "${bootstrap_file}" 'bootstrap.initdb[+].xlogdir' "${PGHA_WALDIR}"
```

Comment:
Just a note that I'll be bumping up the version of `yq` to the latest version (`v3.2.1`) with a PR today, so just want to mention since I'm trying to ensure all `yq` functionality continues to work as expected.  Def like the approach here and eliminating a `yaml` file.